docs/tutorials/autogluon-cloud.md

```diff
@@ -1,2 +1,172 @@
+
+```{.python}
+cloud_predictor.cleanup_deployment()
+```
+
+## Batch Inference
+When minimizing latency isn't a concern, then the batch inference functionality may be easier, more scalable, and more appropriate.
+To do a batch inference,
+
+```{.python}
+cloud_predictor.predict(
+    'test.csv',  # can be a dataframe as well and the results will be stored in s3 bucket
+    instance_type="ml.m5.2xlarge",  # Checkout supported instance and pricing here: https://aws.amazon.com/sagemaker/pricing/
+    wait=True  # Set this to False to make it unblocking call
+)
+cloud_predictor.download_predict_results(
+    job_name=None  # download the most recent finished batch inference results to your local machine. Specify the job name to download a specific batch inference job's results.
+    save_path="PATH"  # If not specified, CloudPredictor will create one.
+)
+```
+
+## Retrieve CloudPredictor Info
+
+To retrieve general info about a `CloudPredictor`
+
+```{.python}
+cloud_predictor.info()
+```
+
+It will output a dict looking similar to this:
+
+```{.python}
+{
+    'local_output_path': '/home/ubuntu/XXX/demo/AutogluonCloudPredictor/ag-20221111_174928',
+    'cloud_output_path': 's3://XXX/tabular-demo',
+    'fit_job': {'name': 'ag-cloudpredictor-1668188968-e5c3',
+    'status': 'Completed',
+    'framework_version': '0.6.1',
+    'artifact_path' 's3://XXX/tabular-demo/model/ag-cloudpredictor-1668188968-e5c3/output/model.tar.gz'},
+    'recent_transform_job': {'name': 'ag-cloudpredictor-1668189393-e95c',
+    'status': 'Completed',
+    'result_path': 's3://XXX/tabular-demo/batch_transform/2022-11-11-17-56-33-991/results/test.csv.out'},
+    'transform_jobs': ['ag-cloudpredictor-1668189393-e95c'],
+    'endpoint': 'ag-cloudpredictor-1668189208-d23b'
+}
+```
+
+## Convert the CloudPredictor to a Local AutoGluon Predictor
+You can easily convert the `CloudPredictor` you trained on SageMaker to your local machine as long as you have the same version of AutoGluon installed locally.
+
+```{.python}
+local_predictor = cloud_predictor.to_local_predictor(
+    save_path="PATH"  # If not specified, CloudPredictor will create one.
+)  # local_predictor would be a TabularPredictor
+```
+
+`to_local_predictor()` would underneath downlod the tarball, expand it to your local disk and load it as a corresponding AutoGluon predictor.
+
+## Note on Image Modality
+If your training and inference tasks involve image modality, you need to
+1. Make sure your column contains **absolute paths** to the images
+2. provide argument `image_column` as the column name containing image paths to `CloudPredictor` fit/inference APIs.
+
+```{.python}
+cloud_predictor.fit(..., image_column="IMAGE_COLUMN_NAME")
+cloud_predictor.predict_real_time(..., image_column="IMAGE_COLUMN_NAME")
+cloud_predictor.predict(..., image_column="IMAGE_COLUMN_NAME")
+```
+
+## Supported Docker Containers
+`autogluon.cloud` supports containers starting AutoGluon Deep Learning Containers 0.6.0.
```

Comment:
I had to re-read this line a couple times. Maybe

> `autogluon.cloud` supports AutoGluon Deep Learning Containers version 0.6.0 and newer.

Reply:
Thanks! This is much better